src/main/kotlin/com/terraformation/backend/documentproducer/db/VariableValueStore.kt

```diff
@@ -795,4 +816,24 @@ class VariableValueStore(
     return dslContext.fetchValue(VARIABLE_VALUES.PROJECT_ID, VARIABLE_VALUES.ID.eq(valueId))
         ?: throw VariableValueNotFoundException(valueId)
   }
+
+  private fun notifyForReview(
+      projectId: ProjectId,
+      values: List<ExistingValue>,
+  ) {
+    val valuesByDeliverables =
+        values.groupBy {
+          val variablesRow =
+              variablesDao.fetchOneById(it.variableId)
+                  ?: throw VariableNotFoundException(it.variableId)
+          variablesRow.deliverableId
+        }
+
+    valuesByDeliverables.keys.forEach { deliverableId ->
+      val thisValues = valuesByDeliverables[deliverableId]!!.associate { it.variableId to it.id }
```

Comment:
If the variable is a list, there will potentially be multiple values with the same variable ID, and items with lower list positions may have been edited after ones with higher positions. This should probably be picking the highest value for each variable.